pkgs/by-name/sh/shadps4/package.nix

```
@@ -127,15 +123,17 @@ stdenv.mkDerivation (finalAttrs: {
       $out/bin/shadps4
   '';
 
-  passthru.updateScript = unstableGitUpdater {
-    tagFormat = "v.*";
-    tagPrefix = "v.";
+  passthru = {
+    tests.openorbis-example = nixosTests.shadps4;
```

```suggestion
    tests = lib.optionalAttrs stdenv.hostPlatform.isLinux {
      inherit (nixosTests) shadps4;
    };
```
This makes sure OfBorg won't evaluate and try to build the test on platforms that don't support it (specifically Mac :p)

---

The test already sets a `meta.platforms`, isn't that fine?

```
       error: Package ‘vm-test-run-shadps4-openorbis-example’ in /private/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/macstadium-m1-44911207/nixos/lib/testing/run.nix:42 is not available on the requested hostPlatform:
         hostPlatform.config = "x86_64-apple-darwin"
         package.meta.platforms = [
           "x86_64-linux"
         ]
         package.meta.badPlatforms = [ ]
       , refusing to evaluate.
```

Would prolly just make sure that the test's platform is always limited to Linux, in case Darwin support gets added to the package:

```diff
   meta = {
-    inherit (pkgs.shadps4.meta) maintainers platforms;
+    inherit (pkgs.shadps4.meta) maintainers;
+    # Linux VM test
+    platforms = lib.intersectLists lib.platforms.linux pkgs.shadps4.meta.platforms;
  };
```

---

> The test already sets a meta.platforms, isn't that fine?

In that case, yeah. This is actually the first time I've seen that 😆